components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/RequestObjectValidatorImpl.java

```diff
@@ -163,7 +163,8 @@ private void checkFapiMandatedParams(RequestObject requestObject) throws Request
     protected boolean isValidAudience(RequestObject requestObject, OAuth2Parameters oAuth2Parameters) throws
             RequestObjectException {
 
-        String tokenEPUrl = getTokenEpURL(oAuth2Parameters.getTenantDomain());
+        String tokenEPUrl = oAuth2Parameters.isMtlsRequest() ? OAuth2Util.OAuthURL.getOAuth2MTLSTokenEPUrl() :
```

Comment:
Don't we need to pass tenant Domain for MTLS token endpoint URL

Reply:
This is already handled at [1]. Tested by creating a tenant user

[1] https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java#L1749-L1756